README.md

```diff
@@ -68,11 +68,11 @@ $ npm run watch
 
 ### To hack cozy-proxy using the cozy vagrant
 
-- Forward a new port from the virtual machine (for example: `config.vm.network :forwarded_port, guest: 9104, host: 9104` in file Vagrantfile)
-- Go in the shared folder `cd /vagrant` and `cd your-cozy-proxy-folder`
-- `rm -rf node_modules/bcrypt && npm install`
-- Launch cozy-proxy `PORT=9104 HOST="0.0.0.0" npm run watch`
-- You can now access the hacked proxy on `http://localhost:9104` with your navigator
+- Forward cozy-home application port from the virtual machine: `config.vm.network :forwarded_port, guest: 9103, host: 9103` in file Vagrantfile
```

Comment:
You are right. I tought it was in the default vagrantfile, but it's not the case